package.json

```diff
@@ -5,21 +5,25 @@
     "ng": "ng",
     "start": "concurrently \"ng serve\" \"yarn build-content:watch\"",
     "build": "ng build",
-    "build:ssr": "ng build --prod && ng run angular-checklist:server -- --output-hashing=none",
-    "prerender": "npm run build:ssr && ts-node prerender.ts",
-    "serve:ssr": "npm run prerender && http-server dist/browser",
+    "#build:ssr": "ng build --prod && ng run angular-checklist:server -- --output-hashing=none",
+    "#prerender": "npm run build:ssr && ts-node prerender.ts",
+    "#serve:ssr": "npm run prerender && http-server dist/browser",
+    "dev:ssr": "ng run angular-checklist:serve-ssr",
+    "serve:ssr": "node dist/angular-checklist/server/main.js",
+    "build:ssr": "ng build && ng run angular-checklist:server -- --output-hashing=none",
+    "prerender": "ng run angular-checklist:prerender",
     "test": "ng test",
     "lint": "ng lint",
     "e2e": "cypress open",
     "build-content": "ts-node ./tools/build-checklist.ts",
     "build-content:watch": "nodemon",
-    "ci": "npm run build-content && ng build --prod",
-    "format:base": "prettier \"{src,tools,cypress}/**/*{.ts,.js,.json,.html}\"",
+    "ci": "npm run build-content && ng build",
+    "format:base": "prettier \"{src,tools,cypress}/**/*{.ts,.js,.json,.scss,.html}\"",
     "format:check": "npm run format:base -- --list-different",
     "format:fix": "npm run format:base -- --write"
   },
   "engines": {
-    "node": "11.15.0"
+    "node": "12.22.7"
```

Comment:
you can use node 16 as it is supported by V13

Reply:
@santoshyadavdev  Thats really good. I struggled a lot with v12 due to lack of **Apple M1** support. Will update the change